src/main/java/net/greatstart/controllers/UserController.java

```diff
@@ -23,11 +29,19 @@
 public class UserController {
     private UserService userService;
     private UserProfileMapper userMapper;
+    private SecurityService securityService;
+    private PasswordEncoder passwordEncoder;
+    private UserValidationService userValidationService;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "userValidationService" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1068)